Dockerfile

```diff
@@ -68,15 +68,34 @@ LABEL org.label-schema.schema-version="1.0" \
 # Docker and docker-compose are always required but take a while to install so
 # they are to be kept at a lower layer for caching.
 RUN apk add --no-cache --virtual .toolchain \
-    python3-dev libffi-dev openssl-dev build-base \
+    python3-dev \
+    py3-pip \
+    libffi-dev \
+    openssl-dev \
+    build-base \
  && apk add --no-cache docker-cli python3 \
  && rm -f /usr/bin/dockerd /usr/bin/docker-containerd* \
  && pip3 install "docker-compose>=1.24.0,<1.25.0" asciinema \
  && apk del .toolchain \
  && rm -rf ~/.cache
 
 # Misc tools required for scripts.
-RUN apk add --no-cache git openssh tree util-linux jq nss-tools multitail ca-certificates highlight libintl entr postgresql-client task bash \
+RUN apk add --no-cache \
+  git \
```

Comment:
As above

Reply:
These dependencies and others where added in #442 so can be removed from this change set, should make rebasing easier too as it appears this has a conflict with the latest master now.